app_grim_reefer/debug.conf

```diff
@@ -16,6 +16,18 @@ CONFIG_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_UART_CONSOLE=y
 
+
+
```

Comment:
Just wipe these extra newlines

Reply:
:grimacing: sry the update saying approved came in before i realized the comment. Will fix for next PR.